pages/linux/export.md

```diff
@@ -18,3 +18,7 @@
 - Append a pathname to the environment variable `PATH`:
 
 `export PATH=$PATH:{{path/to/append}}`
+
+- Display a list of all exported variables and functions:
```

Comment:
```suggestion
- Display all exported variables and functions in a executable format:
```
How about this?

Reply:
TBH I just copied it from `export --help` so I'm not _super_ picky about the specific message. However, I'm a bit of a noob, so the "list"  part makes sense to me where the "executable" part is honestly a bit confusing... but my Bash skills are hardly up to par.

Reply:
I'm inclined to agree with the `list of` version.

The current version is much more readable, I vote to keep as is.

Reply:
I actually prefer @vitorhcl's suggestion. It's a more accurate description because it indicates to the user that you can use the output to reproduce the environment. Also, <s>a</s> an.

Reply:
Its been a while since I contributed, aren't these pages meant to be accessible to everyone, to a new user, what is an `executable format`?

Reply:
@arcanemachine  This is my feeling as well TBH.

Reply:
Yeah, I agree that the wording is a bit confusing, I accept a better one. But I think it is important to leave this clear in some way.

Reply:
> to a new user, what is an `executable format`?

Hmm fair.